<launch>

<rosparam command="load" file="$(find ethercat_demo)/param/ethercat_demo.yaml" />
<param name="ethercat_interface" type="string" value="enxb827eb1cd0b8" />

<node name="joy_node" pkg="joy" type="joy_node" />
<node name="joy_teleop" pkg="joy_teleop" type="joy_teleop.py" />
<node name="geometry_twist_to_velocity" pkg="ethercat_demo" type="geometry_twist_to_velocity" />

</launch>
